tests/series/areaSpec.js

```diff
@@ -18,12 +18,12 @@ describe('area', function() {
         container.datum(data)
             .call(area);
 
-        expect(xValueSpy.calls.count()).toEqual(data.length);
-        this.utils.verifyAccessorCalls(xValueSpy, data);
-
-        // the defined call also invokes the y value accessors,
+        // the defined call also invokes the value accessors,
         // therefore they are invoked twice for each data point
 
+        expect(xValueSpy.calls.count()).toEqual(data.length * 2);
```

Comment:
As per the comment I changed defined to also check x/x0. It's a change to what we had but it seems the logical behaviour.